routes/matches/matches.js

```diff
@@ -329,14 +329,19 @@ router.get("/", async (req, res, next) => {
     let isAscending = req.query?.asc == null ? false : req.query.asc;
     let sql = 
       "SELECT mtch.id, mtch.user_id, mtch.server_id, mtch.team1_id, mtch.team2_id, mtch.winner, mtch.team1_score, " +
```

Comment:
Will these joins not fail if a team gets deleted? I would much rather have a check to see if `is_pug` (as teams are removed after a pug has finished). is enabled in the first query and just use the default statement, otherwise maybe use a join.

Also, try not to bring in everything within a match either. If you need more information, you can just make an API call specific to that route, such as map stats. The whole point of the routes is trying to make the calls as independent as possible.